### PR TITLE
Staging/schedule

### PR DESCRIPTION
**Pull request by Aitori**
Finished most of the schedule page save for the drop down with some ugly buttons but will update, need advice on designs

### CODE REVIEW

File: src/components/NavBar/index.jsx [hunk 1]
```diff
@@ -7,11 +7,11 @@ import logoWhite from 'assets/logo_white.svg';
 import menu from 'assets/icons/menu.svg';
 
 const links = [
-  // { text: 'Schedule', to: '/schedule' },
-  // { text: 'Maps', to: '/maps' },
-  // { text: 'Prizes', to: '/prizes' },
-  // { text: 'Travel', to: '/travel' },
-  // { text: 'Projects', to: '/projects' },
+  { text: 'Schedule', to: '/schedule' },
```
Comment (brianstrauch) [inline src/components/NavBar/index.jsx:10]: Make sure you merge with dev. The only links that should be uncommented are /schedule, /maps, and /mentors

**Review by brianstrauch — CHANGES_REQUESTED** [review]
Looks great! Make sure you're filtering out events that happen before Friday or after Sunday. Also there's an event on Sunday that says 0:00pm haha

**Comment by brianstrauch** [comment]
One more thing, the scenery should overlap the billboard on mobile

Screen Shot 2020-02-06 at 10 23 35 AM

**Review by brianstrauch — CHANGES_REQUESTED** [review]
Looks good but doesn't scroll for some reason

**Comment by Aitori** [comment]
fixed the scroll issue, there was a div on top covering the entire thing but I shortened it, also removed the scroll bar if its not needed

**Review by brianstrauch — APPROVED** [review]

**Pull request merged**